test/integration/options.spec.js

```diff
@@ -346,6 +346,26 @@ describe('options', function() {
           done();
         });
       });
+
+      it('fails if no --grep', function(done) {
+        args = ['--invert'];
+        runMocha(
+          'options/grep.fixture.js',
+          args,
+          function(err, res) {
+            if (err) {
+              done(err);
+              return;
+            }
+            expect(res, 'to satisfy', {
+              code: 1,
+              output: /fgrep/
```

Comment:
Is this the error message? If so can we cover more of the string here? Be really good to be able to check error code (if possible)

Reply:
Using `runMocha` means we're scraping output from stdout|stderr -- we should be able to see `Error.name` and `Error.message`. `Error.code` shouldn't be output if that's what you're asking for.